lib/utils/atomical-operation-builder.ts

```diff
@@ -897,7 +897,7 @@ export class AtomicalOperationBuilder {
     */
     addCommitChangeOutputIfRequired(extraInputValue: number, fee: FeeCalculations, pbst: any, address: string) {
         const totalInputsValue = extraInputValue + this.getTotalAdditionalInputValues();
-        const totalOutputsValue = this.getTotalAdditionalOutputValues() + fee.revealFeePlusOutputs;
+        const totalOutputsValue = fee.revealFeePlusOutputs;
```

Comment:
For semantics, it might be better to be:
```suggestion
        const totalOutputsValue = fee.revealFeeOnly + this.getTotalAdditionalOutputValues();
```

Reply:
Yes.
Perhaps the code here needs to be refactored because there is a fair amount of redundant code.

Reply:
> Yes. Perhaps the code here needs to be refactored because there is a fair amount of redundant code.

That would be another story.